ruby-challenges.rb

```diff
@@ -3,32 +3,131 @@
+# Since we are comparing whether a word is spelled the same forward or backward, we need to use a conditional statement that will evaluate each character of a word that is spelled the same backward or forward. At the end of the conditional statment, we will need to enter and "end" to close the code block.
+# We need to create a string interpoloation to output the word and statement whether a word is spelled the same forward or backward. The string interpolation will output a statement that says whatever word is entered, it will evaluate if the word is spelled the same forward or backward, therefore it will output a statement that says it is spelled the same forward or backward. Using quotations at the beginning of each string interpolation will recognize and output a string (statement in quotations).
+# In the string interpolation, the "str" parameter must be enclosed with a hash and curly braces - #{str} - to enter any word in a string.
+# To evaluate if a word is spelled the same forward, we need to use a class of logical operators (==) and logical 'OR' operator ( || ) that compares each character of the word string to the string and will take in any word that is spelled the same or not spelled the same, and use a method to copmare whether any character is uppercase (.upcase) or lowercase (.downcase).
+# To evaluate if a word is spelled the same backward, we need to use a '.reverse' method that compares each character of the word string to the string and will take in any word that is spelled the same backwards or not spelled the same backwards.  
+# Return an "end" to close the method code block. If any of the above conidtionals are not met, the logic will state that the word is not a palindrome. 
+# We can use 'p' to print out a statement for each by calling on the original method ('palindrome?') together with the given test variables string name (palindrome_tester1, palindrome_tester2, palindrome_tester3) to give us an output with all of the words that are palindrome or not palindrome words.
+
+# Given strings:
+# Input: 'Racecar' ("takes in..")
+# Output: 'Racecar is a palindrome'
+
+# Input: 'LEARN'
+# Output: 'LEARN is not a palindrome'
+
+# Input: 'Rotator'
+# Output:'Rotator is a palindrome'
+
+
 palindrome_tester1 = 'Racecar'
 # Expected output: 'Racecar is a palindrome'
 palindrome_tester2 = 'LEARN'
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome? (str)
+  if str == str.reverse || str == str.upcase.reverse
+    "#{str} is a palindrome"
+  elsif str.downcase == str.downcase.reverse
+    "#{str} is a palindrome"
+  else 
+    "#{str} is not a palindrome"
+  end
+end
+
+p palindrome?(palindrome_tester1)
+p palindrome?(palindrome_tester2)
+p palindrome?(palindrome_tester3)
+# "Racecar is a palindrome"
+# "LEARN is not a palindrome"
+# "Rotator is a palindrome"
```

Comment:
In Ruby, adding a `?` after the method name indicates to developers that the method will return a Boolean. It is not necessary here since this method is actually returning a string. Also, how could you refactor this to only need one `if` and one `else`? Something you can do for a challenge! Good work!